### PR TITLE
amended release notes

### PR DESCRIPTION
**Pull request by robline**

### CODE REVIEW

File: docs/releasenotes.rst [hunk 1]
```diff
@@ -1,23 +1,59 @@
...
+* Sum of parts now accurately matches title (rounding display has been removed to show explicit
+times)
+* Dashboard header styling has been made consistent (hover styling removed and background color
+matched)
+* lessc installs locally rather than globally
+
+*Code Quality*
+
+* Compatibility with Django 1.6 with Python 2.7 or Python 3.3
+* Compatibility with Django 1.7 with Python 2.7 or Python 3.4
```
Comment (daaray) [inline docs/releasenotes.rst]: This needs to be updated for 1.8 and 1.9 support on 2.7. or 3.5; the tox settings are a good metric of what we want to support.

**Comment by robline** [comment]
edited.  thanks.

**Comment by daaray** [comment]
:package:

**Comment by daaray** [comment]
Can you review the README and setup.py for references to Python 3.4 and make them consistent with what is in the rest of this PR (update them all to 3.5)

**Pull request merged**